frontends/web/src/components/accountSelector/accountselector.module.css

```diff
@@ -0,0 +1,5 @@
+.title {
+  font-size: 2rem;
+  font-weight: 400;
+  text-align: center;
+}
```

Comment:
there is also an empty file `frontends/web/src/routes/receive/receive.module.css` but doesn't let me comment there, remove that?

Reply:
oops.. thats quite random. will remove it!